qucs/qucs/dialogs/sweepdialog.cpp

```diff
@@ -46,12 +46,16 @@ mySpinBox::mySpinBox(int Min, int Max, int Step, double *Val, QWidget *Parent)
 using namespace std;
 QString mySpinBox::textFromValue(int Val) const
 {
+  if (Values == NULL) return "";
+
   cout<<"Values + Val"<<*(Values+Val)<<endl;
```

Comment:
Better use `qDebug() << "message";` , so we can silence this on the released code.